browser/src/map/Clipboard.js

```diff
@@ -334,13 +334,9 @@ L.Clipboard = L.Class.extend({
 				);
 			}.bind(this),
 			function(progress) { return progress/2; },
-			function(response) {
+			function() {
 				window.app.console.log('failed to download clipboard using fallback html');
 
-				if (response) {
```

Comment:
Nit: why did you remove this block? This is not something that should happen in practice, so it sounds useful to show a user-visible error in this case?

Reply:
There is a warning via "showInfoModal" few lines below. Looks like it will be shown when the operation fails. So i removed the first one.

Reply:
I see, thanks.